Makefile

```diff
@@ -1,7 +1,6 @@
-BASENAME=$(shell yq -r '.catalog.name' < dabl-meta.yaml 2> /dev/null || yq r dabl-meta.yaml 'catalog.name')
-VERSION=$(shell yq -r '.catalog.version' < dabl-meta.yaml 2> /dev/null || yq r dabl-meta.yaml 'catalog.version')
-SUBDEPLOYMENTS=$(shell yq -r '.subdeployments' < dabl-meta.yaml 2> /dev/null | sed 's/\[//g' | sed 's/\]//g' | sed 's/,//g' \
-	       || yq r dabl-meta.yaml 'subdeployments' | sed 's/\[//g' | sed 's/\]//g' | sed 's/,//g')
+BASENAME=$(shell yq '.catalog.name' dabl-meta.yaml)
+VERSION=$(shell yq '.catalog.version' dabl-meta.yaml)
+SUBDEPLOYMENTS=$(shell yq '.subdeployments[]' dabl-meta.yaml)
```

Comment:
I'm not testing syntax for various versions, because knowing what version you're working with is the entire point of using nix to manage your tool dependencies.